src/data-table/_data-table.scss

```diff
@@ -24,6 +24,7 @@
   white-space: nowrap;
   font-size: 13px;
   background-color: unquote("rgb(#{$color-white})");
+  -webkit-transform: translate3d(0,0,0);
```

Comment:
A comment more like:

`/* Transform in Safari to solve rendering issue.
* Reference: https://github.com/google/material-design-lite/issues/1194
*/`

We want comments to express what the problem is and where it is solving it. Then provide a reference link for more information on the fix and why.